test/fuse/linux/fuse_base.cc

```diff
@@ -172,10 +177,8 @@ PosixError FuseTest::ServerConsumeFuseInit() {
   };
   // Returns a fake fuse_init_out with 7.0 version to avoid ECONNREFUSED
   // error in the initialization of FUSE connection.
-  struct fuse_init_out out_payload = {
-      .major = 7,
-  };
-  auto iov_out = FuseGenerateIovecs(out_header, out_payload);
+  auto iov_out = FuseGenerateIovecs(
+      out_header, *const_cast<struct fuse_init_out*>(out_payload));
```

Comment:
nit: if we pass in `struct fuse_init_out out_payload` object in the function prototype then we don't have to drop const qualification. Although an additional object-copy will be introduced, I think that's fine. I'm OK with either way.